lib/Common/Memory/RecyclerPageAllocator.h

```diff
@@ -40,5 +40,6 @@ class RecyclerPageAllocator : public IdleDecommitPageAllocator
     Recycler* recycler;
 
     bool IsMemProtectMode();
+    virtual char * TryAllocFromZeroPages(uint pageCount, PageSegmentBase<VirtualAllocWrapper> ** pageSegment, PageHeapMode pageHeapFlags) override;
```

Comment:
also looks there's a bug that I checked hasZeroQueuedPages before allocating from zero page list. should check for that only for pending zero page list allocating